README.md

```diff
@@ -21,15 +21,17 @@ npm install
 # Build the front end
 npm run build
 
-
 # Start the application
 npm start
 ```
 
-If you're actively developing, then use `npm run serve` to run the application
+If you're actively developing for the FlowForge core, then use `npm run serve` to run the application
 with automatic rebuilds of the front-end, and restarts of the application when
 it is modified.
 
+If you intend on developing for the additional repositories, please refer to [the docs](https://github.com/flowforge/flowforge/tree/dev-docs#readme) 
```

Comment:
```suggestion
If you intend on developing for the additional repositories, please refer to [the docs](https://github.com/flowforge/flowforge/tree/dev-docs#readme) 
```
Where should this be linked to? Currently it points at the same file but on your feature branch (that will get deleted once this PR is merged).

Reply:
```suggestion
If you intend on developing for the additional repositories, please refer to [the docs](https://flowforge.com/docs/contribute/) 
```

Well spotted. Should be `https://flowforge.com/docs/contribute/`

Reply:
Or should it be `https://github.com/flowforge/flowforge/tree/main/docs/contribute#readme`? 🤔